templates/zerver/help/share-a-message-or-conversation.md

```diff
@@ -0,0 +1,27 @@
+# Sharing a message or conversation
+
+A message or conversation can be shared on Zulip by creating
+ a link to the conversation and sharing the link.
+
+1. To create a link to a conversation, first locate the relevant message.
+
+2. Hover over the message to reveal a {!down-chevron.md!}.
+
+3. Click on the {!down-chevron.md!} and select
+ the **Link to conversation** option from the dropdown.
+
+4. Upon clicking **Link to conversation**, Zulip will scope into just that
+ message's conversation. The keywords **topic:** and **near:** will be added to
+ the search bar above like so:
+
+    ![Example message search](/static/images/help/search-message.png)
+
+Furthermore, the URL in your web browser will now link
```

Comment:
This is breaking the tip and all other lines below. You need to add an indent at line 19.